app/models/site_config.rb

```diff
@@ -31,7 +31,7 @@ class SiteConfig < RailsSettings::Base
     instagram: nil,
     twitch: nil
   }
-  field :twitter_hashtag, type: :string
+  field :twitter_hashtag, type: :string, default: "DEVCommunity"
```

Comment:
shouldn't the pound sign, `#`, be part of the string? After all the variable name is `twitter hashtag`.

Reply:
@rhymes That makes sense! By the way, I've noticed that %23 is used for URL encoding purposes. Would it be appropriate to then use "%23DEVCommunity" as the default value or "#DEVCommunity" and replace # w/ %23 where appropriate?